backend/src/main/java/com/fortunator/api/models/TransactionCategory.java

```diff
@@ -0,0 +1,64 @@
+package com.fortunator.api.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonProperty.Access;
+
+@Entity
+@Table(name = "transaction_categories")
+public class TransactionCategory {
+	
+	@JsonProperty(access = Access.READ_ONLY)
+	@Id
+	@GeneratedValue(strategy=GenerationType.IDENTITY)
+	private Long id;
+	
+	@NotBlank
+	@Size(max = 60)
+	private String name;
```

Comment:
Sugestão: o que acha de colocar uma lógica para limpar o valor que vem em `name`, normalizando para lowercase caso não seja, talvez tbm retirar espaços vazios caso tenha ?

Exemplo: Se a entrada for `Fast food` daria pra transformar para `fastfood` ou `fast_food`

Reply:
Pode ser, a mas qual seria a vantagem? Pq o front teria que formatar dnovo para mostrar.